server/review/queries.ts

```diff
@@ -140,19 +143,19 @@ export const updateReview = async (inputValues, updatePermissions, userData) =>
 		where: { id: inputValues.reviewId },
 		returning: true,
 	})
-		.then((updatedReview) => {
+		.then(async (updatedReview) => {
 			if (!updatedReview[0]) {
 				return {};
 			}
 			const nextValues = updatedReview[1][0].get();
-			const prevStatus = previousReview.status;
+			const prevStatus = (await previousReview)?.status;
 			const wasClosed = prevStatus !== 'closed' && nextValues.status === 'closed';
 			const wasCompleted = prevStatus !== 'completed' && nextValues.status === 'completed';
 			if (wasClosed) {
-				return Promise.all[createClosedThreadEvent(userData, nextValues.threadId)];
+				return Promise.all([createClosedThreadEvent(userData, nextValues.threadId)]);
 			}
 			if (wasCompleted) {
-				return Promise.all[createCompletedThreadEvent(userData, nextValues.threadId)];
+				return Promise.all([createCompletedThreadEvent(userData, nextValues.threadId)]);
```

Comment:
this was realy weird, don't think this ever got called to begin with, but obviously `Promise.all[<randomObject>]` won't work.